src/services/eventLog/index.ts

```diff
@@ -34,27 +34,39 @@ export async function getMaxEventLog(blockNum: number) {
     });
 }
 
-export function webSocketFormat(eventLogs: EventLog[], accounts: string[]) {
+export function webSocketFormat(
+    eventLogs: EventLog[],
+    accounts: string[],
+    lastEventLogId: number,
+    lastCheckedBlock: number
+) {
     const parsedTraces = eventLogs.map(({ data, ...tx }) => ({
         ...tx,
         data: JSON.parse(data.toString('utf8')),
     }));
 
     return parsedTraces
+        .filter((tx) => {
```

Comment:
tx is of type any because of `JSON.parse` above. Let's create a type + type guard for trace.data